### ISSUE TITLE
'Initialize' and 'New' bugs in ;first-aid and ;combat-trainer

### ISSUE THREAD

**Issue by Missah** [rpherbig/dr-scripts#2008]
I have first-aid set up in my BEFORE hunting settings.

When it gets to the last study command:

```
[first-aid]>study my guide

You begin studying the Celpeze chart, gradually absorbing the knowledge contained within.
Roundtime: 20 seconds.

--- Lich: error: private method `empty_hands' called for #<Scripting::EquipmentManager:0xe62b300>
	first-aid:30:in `initialize'
	first-aid:66:in `new'

--- Lich: first-aid has exited.
```

***Guide has not been stowed at this point.  Stays in hands - get to hunting ground:

```
--- Lich: go2 has exited.

[hunting-buddy]>search

You search around for a moment.
Roundtime: 3 sec.
You don't find anything of interest here.

[hunting-buddy]>say I hope no one else is here, I really want to kill something.

You say, "I hope no one else is here, I really want to kill something."

[hunting-buddy: ***STATUS*** beginning hunt '["d0"]' for '60' minutes]

--- Lich: combat-trainer active.

--- Lich: error: wrong number of arguments (1 for 0)
	combat-trainer:2008:in `initialize'
	combat-trainer:2008:in `new'
```

Where I proceed to stand and get pummeled.

Hope this can be fixed, thanks!

**Comment by rpherbig** [comment]
Can you post or link to your YAML file?

**Comment by rpherbig** [comment]
My initial guess is that you have an old version of `equipmanager`, though I'm not sure how that would happen.

**Comment by Missah** [comment]
Also getting this error at login:

--- Lich: repository has exited.

-- Lich: repository active.
--- Lich: dependency active.
--- Lich: dr-autovote active.
--- Lich: dependency is already running (use ;force [scriptname] if desired).
--- Lich: afk active.
--- Lich: error: undefined local variable or method `custom_require' for #<Scripting:0x24dc6c8>
	afk:5:in `script'
	C:/Users/Rose/Documents/lich/lich.rbw:2546:in `eval'
--- Lich: afk has exited.
--- Lich: circlemonitor active.
--- Lich: error: undefined local variable or method `custom_require' for #<Scripting:0x2660d00>
	circlemonitor:7:in `script'
	C:/Users/Rose/Documents/lich/lich.rbw:2546:in `eval'
--- Lich: circlemonitor has exited.

Here is link to my yaml:

https://pastebin.com/zaMfDet8

Thanks!

**Comment by rpherbig** [comment]
It looks like dependency tried to start up twice there. And it looks like
afk is using the lich auto start, not dependency auto start

**Comment by rcuhljr** [comment]
The second set of errors are all caused by adding things to `;autostart` instead of using `;e autostart('name')`

**Comment by Missah** [comment]
so...how do I fix?

**Comment by Missah** [comment]
I asked how to fix too soon!  I have removed all the autostarts from the ;autostart and added them to ;e autostart list -- which for me, was very scary.  However, I am now logging in with no errors.

I remember that I added the scripts to ;autostart when I was having issues playing a toon in GSIV - which cleared up the issue, but now I'm no longer playing GS.

I haven't tried combat yet, but will close this issue per both your indications that the errors I was receiving were due to the autostart. 

Thank you so much, as always, for your help!
